swig/python2/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+if(ENABLE_PYTHON)
+	FIND_PACKAGE(SWIG REQUIRED)
```

Comment:
only use findpackage in the root CMakeLists. use the same convention through all cmake files, don't mix CAPSLOCK and CamelCase.